### PR TITLE
Report Komodo release used

### LINKED ISSUE #305: komodo version
Internally in Equinor, https://github.com/equinor/komodo is used to deploy software. If we detect that komodo is used, should we add a comment in generated portable app which komodo release it was built with?

### PR DESCRIPTION
**Pull request by anders-kiaer**
Closes #305.

### CODE REVIEW

File: webviz_config/_build_webviz.py [hunk 1]
```diff
@@ -133,3 +136,21 @@ def run_webviz(args: argparse.Namespace, build_directory: pathlib.Path) -> None:
                 f"{terminal_colors.END}"
             )
             raise excep
+
+
+def _store_komodo_release(build_directory: pathlib.Path) -> None:
+    """Internally in Equinor, https://github.com/equinor/komodo is one platform used
+    for software deployment. For portable builds we want, until cloud hosting is common for
+    everyone, mention for the user which specific Komodo release (~= pip list) that was used
```
Comment (asnyv) [inline webviz_config/_build_webviz.py:144]:
```suggestion
    everyone, to mention for the user which specific Komodo release (~= pip list) that was used
```

File: webviz_config/_build_webviz.py [hunk 2]
```diff
@@ -133,3 +136,21 @@ def run_webviz(args: argparse.Namespace, build_directory: pathlib.Path) -> None:
...
+    to build the portable Webviz application.
+
+    This function stores the Komodo release used, if Komodo is used, in a file komodo_release.txt.
+    """
+
+    if "KOMODO_RELEASE" in os.environ and sys.executable.startswith("/prog/res/komodo"):
+        (build_directory / "komodo_release.txt").write_text(
+            "This portable Webviz application was built with Komodo release:\n"
+            f"{os.environ['KOMODO_RELEASE']}\n\n"
+            "Try sourcing this Komodo release if you "
```
Comment (asnyv) [inline webviz_config/_build_webviz.py:154]: Suggest that we explicitly state the required command to source the komodo release (probably best to use the csh alternative)

**Review by asnyv — CHANGES_REQUESTED** [review]
Think we should consider to be even more explicit on what/how to source the correct environment.
Most users will understand what is currently there, but doesn't hurt to add the full command.

**Comment by anders-kiaer** [comment]
Will be moved to https://github.com/equinor/webviz-config-equinor, as it is primarily Equinor that uses Komodo.

**Pull request closed without merging**